xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/LivedataDropdownMenu.vue

```diff
@@ -47,47 +47,42 @@
     <ul class="dropdown-menu dropdown-menu-right">
 
       <!-- Actions -->
-      <li class="dropdown-header">{{ $t('livedata.dropdownMenu.actions') }}</li>
-
-      <li>
-        <!-- Refresh -->
-        <a href="#" @click.prevent="logic.updateEntries()" class="livedata-action-refresh">
-          <XWikiIcon :icon-descriptor="{name: 'repeat'}" /> 
-          {{ $t('livedata.action.refresh') }}
-        </a>
-      </li>
-
+      <li><span class="dropdown-header">{{ $t('livedata.dropdownMenu.actions') }}</span><ul>
+        <li>
+          <!-- Refresh -->
+          <a href="#" @click.prevent="logic.updateEntries()" class="livedata-action-refresh">
+            <XWikiIcon :icon-descriptor="{name: 'repeat'}" /> 
+            {{ $t('livedata.action.refresh') }}
+          </a>
+        </li>
+      </ul></li>
 
       <!-- Layouts -->
-      <li role="separator" class="divider"></li>
-
-      <li class="dropdown-header">{{ $t('livedata.dropdownMenu.layouts') }}</li>
-
-      <!-- Layout options -->
-      <li
-        v-for="layout in data.meta.layouts"
-        :key="layout.id"
-        :class="{
-          'disabled': isCurrentLayout(layout.id),
-        }"
-      >
-        <a href="#" @click.prevent="changeLayout(layout.id)">
-          <XWikiIcon :icon-descriptor="layout.icon"></XWikiIcon>
-          {{ layout.name }}
-        </a>
-      </li>
+      <li><span class="dropdown-header">{{ $t('livedata.dropdownMenu.layouts') }}</span><ul>
+        <!-- Layout options -->
+        <li
+          v-for="layout in data.meta.layouts"
+          :key="layout.id"
+          :class="{
+            'disabled': isCurrentLayout(layout.id),
+          }"
+        >
+          <a href="#" @click.prevent="changeLayout(layout.id)">
+            <XWikiIcon :icon-descriptor="layout.icon"></XWikiIcon>
+            {{ layout.name }}
+          </a>
+        </li>
+      </ul></li>
 
       <!-- Panels -->
-      <li role="separator" class="divider"></li>
-
-      <li class="dropdown-header">{{ $t('livedata.dropdownMenu.panels') }}</li>
-
-      <li v-for="panel in logic.panels" :key="panel.id">
-        <a href="#" @click.prevent="logic.uniqueArrayToggle(logic.openedPanels, panel.id)">
-          <XWikiIcon :icon-descriptor="{name: panel.icon}"/>
-          {{ panel.name }}
-        </a>
-      </li>
+      <li><span class="dropdown-header">{{ $t('livedata.dropdownMenu.panels') }}</span><ul>
```

Comment:
that part could be indented as well.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/3662/commits/3cb1436797723d677f033e7c7000b38115de3d87 :+1: